example/example/ExampleViewController.swift

```diff
@@ -7,11 +7,19 @@
 //
 
 import UIKit
+import AblyRealtime
 
 class ExampleViewController: UIViewController {
 
+    private let API_KEY = "INSERT-YOU-API-KEY-HERE" /* Add your API key here */
```

Comment:
This is missing an important exception notice for the user that they cannot just run this demo without first setting the API KEY, see https://github.com/ably/tutorials/commit/9e06aafb1ee1473d54137a72f3a3f5940c1b9c3e#diff-af4ebf18890510626fe0fab30efeab2cR4 as an example.  If this app starts and the API key is not set up, then perhaps a dialog / modal should be shown to the user stating "You have not set the API key, please configure the API key in ExampleViewController.swift to run this demo"